pkg/apis/ako/v1alpha2/common.go

```diff
@@ -111,3 +111,8 @@ type Service struct {
 	Port      *uint32 `json:"port"`
 	Protocol  *string `json:"protocol"`
 }
+
+type KindNameNamespace struct {
```

Comment:
structure name includes `namespace` so that in future if namespace got included (For CRD support ).